actors/runtime/Cargo.toml

```diff
@@ -63,4 +63,7 @@ min-power-2k = []
 # Lower the minimum power requirement to 2g
 min-power-2g = []
 
+# no collateral for deals (for testing)
+no-collateral = []
```

Comment:
This feature name is a bit ambiguous, as there are several types of collateral in the system. How about zero-provider-deal-collateral?

Reply:
a bit of a mouthful, how about no-provider-collateral?

Reply:
no-provider-deal-collateral it is.